Helper/Data.php

```diff
@@ -258,4 +258,13 @@ public function logError($exception)
             $this->log($exception);
         }
     }
+    
+    public function buildBackendParams($storeId) {
+        return array (
+            'time'            => round(microtime(true) * 1000),
```

Comment:
Changed to use time() instead of microtime() because the time on miliseconds returns float number instead of integer.